### ISSUE TITLE
Create screen object to collect different modes, chunky/planar etc

### ISSUE THREAD

**Issue by Eleas** [Eleas/IFF_ImageReader#15]
We need an object to represent the screen. There are several concerns.

- A screen has a number called width.
- A screen has a number called height.
- A screen has a number called color depth, i.e. how many bits represent each color in RGB.
- A screen can be true color (where the data read is interpreted as its own color value) or palette based (where the value is matched to a lookup palette).
- A screen can be planar or chunky. A planar screen has a number called its bitplanes.
- A planar screen can be EHB.
- A planar screen can be HAM.
- A planar screen can be sliced.

In short, chunky is by far the simpler structure, and we seem to need these classes: Screen (ChunkyScreen, PlanarScreen) and ColorLookup (Palette, Direct).
